### ISSUE TITLE
Collected shows 0 in draft overlay

### ISSUE THREAD

**Issue by Drizzt893** [Razviar/mtgap#282]
Each time I draft Crimson Vow Traditional Draft, It shows collected cards as 0 in the draft overlay.  I draft in order to collect cards I don't own.  I remember this being an occasional problem when I used this program previously (I haven't played arena for 6 months, so this might be an old problem).  In the previous Midnight Hunt Quick Drafts and the current Zendikar Quick Draft, the collection count show up properly.

![Screenshot (70)](https://user-images.githubusercontent.com/57761055/141664685-74777d8d-800d-4f90-be9b-78f37c54f281.png)

**Comment by Razviar** [comment]
Do you have collection synced on site? Is it showing correct values on /collection/ page

**Comment by Drizzt893** [comment]
> 
> 
> Do you have collection synced on site? Is it showing correct values on /collection/ page?

Yes, the website shows my collection, including the VOW cards I own.
